ES/ESum042.xml

```diff
@@ -644,250 +355,425 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           The text was copied by <persName ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName> (see <locus target="#138r"/>). 
+                           Priest <persName ref="PRS13964KeflaMaryam">Kǝfla Māryām</persName> is mentioned as the commissioner on <locus target="#138r"/>, 
+                           <persName ref="PRS13966WalattaKrestos" role="owner">Walatta Krǝstos</persName>, mentioned several times, might be his relative. 
+                        </origin>
+                     </history>
+                     </msPart>
+               
+                     <msPart xml:id="p3">
+                   <msIdentifier/>
+                   <msContents>
+                     <summary/>
+                      
+                      <msItem xml:id="p3_i1">
+                         <locus from="139" to="144v"/>
+                         <title type="complete" ref="LIT4669TaammeraQi">4 miracles of Cyricus</title>
+                         <textLang mainLang="gez"/>
+                         
+                         <msItem xml:id="p3_i1.1">
+                            <locus from="139ra" to="139vb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 1. About a pious king who accept to be a priest in the land of Roḥa</title>
+                            <textLang mainLang="gez"/>   
+                            <incipit xml:lang="gez">
+                                በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ነዋ፡ ጸሐፍነተ፡ አምሪሁ፡ ለቂርቆስ፡ ሰማዕት፡ 
+                                ንስቲተ፡ እምብዙኃ፡ ከመ፡ ኢይኩን፡ ዝንጋዔ፡ ለሰማዕያን፡ ጸሎቱ፡
+                                ወበረከቱ፡ የሀሉ፡ ምስለ፡ ክፍለ፡ ማርያም፡ ለዓለመ፡ ዓለም፡ አሜን። ወሀሎ፡ ፩ንጉሥ፡ ጻድቅ፡ ዘተወክፈ፡ ሢመተ፡ ክህነት፡ እምአፈ፡ 
+                               አምላኩ፡ በሀገረ፡ ሮሐ፡ ቅድስት፡ 
+                            </incipit>
+                      </msItem>
+                         
+                         <msItem xml:id="p3_i1.2">
+                            <locus from="140ra" to="140rb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 2. About a believer in a church dedicated to Qirqos</title>
+                            <textLang mainLang="gez"/>                            
+                         </msItem>
+                         
+                         <msItem xml:id="p3_i1.3">
+                            <locus from="140vb" to="141rb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 3. About a poor monk from a distant country</title>
+                            <textLang mainLang="gez"/>                            
+                         </msItem>
+                                                  
+                         <msItem xml:id="p3_i1.4">
+                            <locus from="141rb" to="141vb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 4. About a monk called Eli (ʾElyās)</title>
```

Comment:
as above

Reply:
[Miracle of St Cyricus (Qirqos): St. Cyricus and the monk ʾElyās](https://betamasaheft.eu/LIT5721MiracleCyriacus)

Reply:
thanks!